docs/Acknowledgement_text.rst

```diff
@@ -0,0 +1,16 @@
+.. note::
+
+   Researchers are requested to acknowledge Mila when publications or reports
+   are published on research that used resources provided by the organization.
+   Our ability to showcase the successful use of our resources builds an
+   important case for long-term and sustainable funding of Mila and our
+   operations. Examples include the use, either in whole or in part, of:
+
+   * Computing, visualization or storage hardware
+   * Software from `<https://github.com/mila-iqia>`_
+   * Staff expertise / consultation
+
+   The exact wording of the acknowledgment may vary, but please ensure that
+   Mila is mentioned. The following can be used as a guideline:
+
+      *"This research was enabled in part by support provided by Mila (mila.quebec)."*
```

Comment:
Do we want this acknowledgment for any kind of resources or computational resources specifically?

Reply:
I think it makes sense to have both? This is largely using the acknowledgement of CC which made sense to me as well: https://alliancecan.ca/en/services/advanced-research-computing/acknowledging-alliance

Reply:
I think in the case of Compute Canada it's quite clear that the resources they provide is compute. Mila on the other hand provides lots of different resources to researchers. If what we mostly need to know if what papers used compute provided by Mila to support extending the cluster then I'd say the acknowledgement for Mila resources needs to be more precise than the one for Compute Canad.

Reply:
I'm not entirely sure why we wouldn't want to receive acknowledgement for software / consultation / general help as well (even if we might not have that much right now) but I've cleaned up the text

Reply:
Indeed, it would also be useful to receive acknowledgement for these services. Maybe the sentence could include all these explicitly, and we could ask the researchers to adapt the sentence to whatever services they benefited from for their research.